README.md

```diff
@@ -10,13 +10,26 @@ database versions, including Oracle.
 
 ## Quickstart
 
-Clone this repository somewhere. Also make sure you have docker and docker-compose installed.
+Clone this repository and ensure you have docker and docker-compose installed.
 
-Ensure that the `DJANGO_PATH` variable points to the root of the Django repo:
+### Environment
+
+You must set the `DJANGO_PATH` environment variable to the path of your local Django installation.
```

Comment:
Maybe `checkout` instead of `installation`? This project is usually run against a local checkout of https://github.com/django/django/ and not an install of the package which doesn't inclue the `/tests` directory anyway?

```
Point the `DJANGO_PATH` environment variable to the path of your local Django checkout.
```